test/integration/session_answers_test.rb

```diff
@@ -0,0 +1,32 @@
+require "test_helper"
+GovukTest.configure
+
+class SessionAnswersTest < ActionDispatch::SystemTestCase
+  setup do
+    Capybara.current_driver = :headless_chrome
```

Comment:
`:headless_chrome` is configured by `GovukTest` and is the [recommended driver documented for that gem](https://github.com/alphagov/govuk_test#usage). Other tests in this app are using `:rack_test` which doesn't seem to work headlessly in a default Rails integration test (`ActionDispatch::SystemTestCase`). So using headless chrome here and then reverting to the app default (`:rack_test`) afterwards.